### PR TITLE
Response body Process effect workaround transplant

### PR DESCRIPTION
**Pull request by drostron**

### CODE REVIEW

File: web/src/main/scala/quasar/api/services/metadata.scala [hunk 1]
```diff
@@ -87,7 +89,7 @@ object metadata {
       InvalidMountNode.invalidMountNodeEncodeJson(_),
       FsNode.fsNodeEncodeJson(_)))
 
-  def service[S[_]](implicit Q: QueryFile.Ops[S], M: Mounting.Ops[S]): QHttpService[S] = {
+  def service[S[_]](implicit S0: Task :<: S, Q: QueryFile.Ops[S], M: Mounting.Ops[S]): QHttpService[S] = {
```
Comment (wemrysi) [inline web/src/main/scala/quasar/api/services/metadata.scala]: We could be a bit more precise here (and elsewhere) and instead require `Catchable[Free[S, ?]]` and leave it up to the caller to figure out how to make it so.

Comment (drostron) [reply web/src/main/scala/quasar/api/services/metadata.scala]: Dig it

**Comment by drostron** [comment]
@wemrysi Ready

**Comment by jedesah** [comment]
So what was the motivation for this change? As `QHttpService.apply` is used in some places to transform an existing `QHttpService`, I'm not convinced putting this logic in this constructor is ideal as we will end with this behavior repeated multiple times.

I think perhaps adding it to `QResponse.streaming` would be more appropriate.

Anyway, I'll be curious to hear the reason for the original change. Hopefully, we can figure this is out early Monday morning as it's sort of blocking the update to latest Quasar in SB

**Pull request merged**